### DISCUSSION TITLE
A new boilerplate project

### DISCUSSION THREAD

**Discussion by krishnapollu** [webdriverio/webdriverio#10888] [Community]
Hello All,

Please check out my new boilerplate project. Hope this might be helpful to the community.
[wdio-cucumber-poc](https://github.com/krishnapollu/wdio-cucumber-poc)

I have considered the below test requirements in this project

- Cucumber based E2E Tests
- Mocha based Data Driven Tests
- Web only Tests - in Local as well as cloud platforms
- Mobile Only tests - local as well as remote cloud emulators (or devices)
- Web + Mobile Tests - Multiremote - local as well as cloud platforms
- Multiple Reports integrated
- Test Data handled globally as I need all the data created to be written back to a file post test exexution
- Github workflow to run the test and upload the allure report

**Comment by christian-bromann** [comment]
Awesome @krishnapollu 🎉 feel free to raise a PR and add it to https://github.com/webdriverio/webdriverio/blob/main/website/docs/BoilerplateProjects.md to have it show on the website.
